linter/whitelists/en.txt

```diff
@@ -6,6 +6,7 @@ AntiSpam
 Assets
 Assistant
 Avatar
+Barcode
```

Comment:
A `Barcode` sounds very generic to me too. Imho that should go to `default.txt` as well.